controllers/admin/AdminStoresController.php

```diff
@@ -599,6 +599,6 @@ protected function adaptHoursFormat($value)
     {
         $separator = array_fill(0, count($value), ' | ');
 
-        return array_map('implode', $value, $separator);
+        return array_map('implode', $separator, $value);
```

Comment:
You can't do this as we support PHP 7.1 => 7.4. You must check the version of PHP.

Reply:
🤯  what they SWITCHED the argument order in php7.4

Reply:
https://www.php.net/manual/en/function.implode.php

Before php 7.4, both orders are supported, as of PHP 7.4, only one order is supported (the other order is deprecated)

Reply:
@matthieu-rolland Do you mean that if you run 1.7.8 with php7.2, php is going to be able to find out "what is glue, what is array" because of the type?

Reply:
Of course, I tested that it still works with an anterior PHP version (7.3)

Reply:
yep ! I tested in php 7.3